### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by BobinMathew**
**Related Issue(s):** #
Issue #385

**Description:**
Created a CODE_OF_CONDUCT.md that points to  Code of Conduct in the STAC spec repo. This makes the code of conduct consistent with the STAC repo's code of conduct

**PR Checklist:**

- [ ] Code is formatted (run `scripts/format`)
- [ ] Tests pass (run `scripts/test`)
- [ ] This PR maintains or improves overall codebase code coverage.
- [ ] Changes are added to the [CHANGELOG](https://github.com/stac-utils/pystac/blob/develop/CHANGELOG.md). See [the docs](https://pystac.readthedocs.io/en/latest/contributing.html#changelog) for information about adding to the changelog.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Stac-Utils Community Code of Conduct
+
+Stac-Utils follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).
```
Comment (gadomski) [inline CODE_OF_CONDUCT.md:3]:
```suggestion
# pystac Community Code of Conduct

pystac follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).
```

**Review by duckontheweb — COMMENTED** [review]
Thanks for contributing this @BobinMathew, and for recognizing the gap.

I think it might be good to at least include the [Our Pledge](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md#our-pledge) portion of the Code of Conduct in full in this document and then reference the STAC Code of Conduct for details on the specific standards. It seems important to make a clear statement in this repo about the kind of environment that are hoping to foster so that newcomers know it's something that's important within the community.

Also, the STAC spec Code of Conduct lists stac-spec-admins@googlegroups.com as the contact to report abuses and unacceptable behavior. Do we need a different email, or can we use that one here (@cholmes)?

**Comment by BobinMathew** [comment]
- [X] Included Pledge
- [ ] contact details pending...

**Comment by duckontheweb** [comment]
@BobinMathew We are going to use the STAC Project Steering Committee email (stac-psc@radiant.earth) as our point of contact. It sounds like the `stac-spec` repo may also update their Code of Conduct to use this email at some point. 

I think it would be good to include that information in this repo as well to make it as easy as possible for someone to report a violation or other problem.

**Comment by BobinMathew** [comment]
* [x]  Included Pledge
* [x]  contact details added

**Review by duckontheweb — APPROVED** [review]
This looks good to me, thanks for contributing this @BobinMathew!

I'd like to get approval from at least @cholmes and @lossyrob as well and give other folks a chance to comment if they'd like.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# pystac Community Code of Conduct
```
Comment (lossyrob) [inline CODE_OF_CONDUCT.md:1]: Spelling elsewhere is PySTAC when used in this sort of context

```suggestion
# PySTAC Community Code of Conduct
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## Our Pledge
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [stac-psc-admins](mailto:stac-psc@radiant.earth). All
```
Comment (lossyrob) [inline CODE_OF_CONDUCT.md:9]:
```suggestion
reported by contacting the [STAC Project Steering Committee](mailto:stac-psc@radiant.earth). All
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+pystac follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).
```
Comment (lossyrob) [inline CODE_OF_CONDUCT.md:19]:
```suggestion
PySTAC follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).
```

**Review by lossyrob — CHANGES_REQUESTED** [review]
Made some suggestions for small changes

**Review by lossyrob — APPROVED** [review]

**Comment by BobinMathew** [comment]
Thanks everyone 😀

**Review by cholmes — APPROVED** [review]

**Comment by duckontheweb** [comment]
Thanks again for the contribution @BobinMathew!

**Pull request merged**